examples/dynamo/converter_overloading.py

```diff
@@ -0,0 +1,210 @@
+    priority=torch_tensorrt.dynamo.conversion.ConverterPriority.HIGH,
+)
+
+# %%
+# For the decorator defining a converter, there is one required argument and a few optional ones.
+# All converters need a target operator they will run against, the idea being that when there is an instance of ``torch.ops.aten.gelu.default`` in the graph, this converter will be called.
+#
+# Following the target operator, you can provide additional metadata that defines the capabilities of the converter and the priority of the converter verses other possible converters for the target in question
+#
+# The primary tool for defining the capabilities of a converter is the ``capability_validator`` argument,
+# which is a lambda function that takes a specific node in the graph as well as the user compilation settings and returns a boolean indicating if the converter can be used for that node.
+# This validator function gets run prior to the graph partitioning phase against each instance of the converter target op. Nodes where there are no converters with validators that pass during this phase, will be executed in PyTorch at runtime.
+# This is useful for cases where you want to use a custom converter only in specific cases, like in our case where we only want to use our converter when ``approximate == "tanh"``.
+#
+# Distinct to the validator is the ``supports_dynamic_shapes`` argument, which is a boolean indicating if the converter can be used in cases where the input shapes are dynamic.
+# If this is set to ``False``, in cases where the inputs provided by the user are dynamic, this converter will be disabled. If there are no alternatives that support dynamic shape, the operation will be run in PyTorch.
+#
+# Finally there is the ``priority`` argument, which is an enum from the ``torch_tensorrt.dynamo.conversion.ConverterPriority`` class that defines the priority of the converter. The two options are ``HIGH`` and ``STANDARD``.
+# Converters registered with ``STANDARD`` will be appended to the converter list for a given operation, while converters registered with ``HIGH`` will be prepended to the list.
+# Canidate converters are evalated for their suitablity in this priority order and the first converter that passes the validator is used.
+
+
+# %%
+# Converter Implementation
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The converter function itself takes the following arguments: the current conversion context, the target operator, the arguments to the target operator, the keyword arguments to the target operator, and the name of the target operator.
+# Arguments can either any of python primitives, ``torch.Tensor``, ``np.Arrays`` or ``ITensor`` objects.
+# The converter function should return the outputs of the target operator in terms of TensorRT ``ITensor`` primarily.
+#
+# Since Torch-TensorRT covers the core ATen opset, it has already abstracted many of the common low-level operations into helper functions that can be used to build up the TensorRT network.
```

Comment:
Do we need to mention where users can find the ATen schema so that they can get to know the args' positions and types?